src/librustc_mir/constness.rs

```diff
@@ -1,16 +1,28 @@
-use crate::hir;
```

Comment:
Please move this file to `src/librustc_mir/const_eval/fn_queries.rs` and reexport the functions in the `const_eval` module